### ISSUE TITLE
The following change cause spring start failed when use spring directly in $LOAD_PATH

### ISSUE THREAD

**Issue by zw963** [rails/spring#435]
I use spring independent with specify project, e.g. spring not installed in bundle.
I thought many developer use like this, it just a personal preference.

in fact, I just drop spring code into some ruby `$LOAD_PATH`  directory, it worked.

When i update to spring 1.4, following change broken this feature.

https://github.com/rails/spring/blob/master/lib/spring/client/run.rb#L68-L72

it will result in following error. 
```sh
/home/zw963/.rvm/rubies/ruby-2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/dependency.rb:315:in `to_specs': Could not find 'spring' (= 1.4.0) among 144 total gem(s) (Gem::LoadError)
Checked in 'GEM_PATH=/home/zw963/.rvm/gems/ruby-2.2.3@payment:/home/zw963/.rvm/gems/ruby-2.2.3@global', execute `gem env` for more information
	from /home/zw963/.rvm/rubies/ruby-2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/dependency.rb:324:in `to_spec'
	from /home/zw963/.rvm/rubies/ruby-2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_gem.rb:64:in `gem'
	from -e:1:in `<main>'
```
seem like it caused by gem version's require can not found spring.

**Comment by grosser** [comment]
I don't think that's a supported usecase ... can you use `gem install spring` instead ?

**Comment by grosser** [comment]
what is the last version that worked for you ... 1.3.6 should also have that code from PR https://github.com/rails/spring/pull/402

**Comment by zw963** [comment]
Thanks.
My last working version is spring 1.3.3, But now I use 1.4.0, I just restore this
piece of code from 1.3,3,  it worked fine for me. 

I think this should be a supported usecase, ; ), so I create a issue here. 

anyway, my all project use only one same version spring, it avoid I have to 
add this gem(and so many fancy gems like spring) to rvm @global namespace
when update ruby, this is painful, and waste of disk usage, I use those gem
anywhere anyway.

**Comment by jonleighton** [comment]
It is supported to [use Spring outside of your project bundle](https://github.com/rails/spring#use-without-adding-to-bundle) but you must install it as a gem. It is not supported to use spring by "dropping it in the load path".

**Comment by zw963** [comment]
Thanks.
Although almost gem I used is support "dropping it in the load path".  ; )

**Comment by sonalkr132** [comment]
Hey, can this be close this if this feature is not supported and there are no plans to support it?

**Comment by zw963** [comment]
I think it shoud be close by maintainer.

**Comment by zw963** [comment]
I have to say, altough not support, But this problem is fixed in spring 1.7.2 which bundle in rails 5.
